technology/cassandra/src/main/java/com/github/mizool/technology/cassandra/ClusterProducer.java

```diff
@@ -0,0 +1,61 @@
+package com.github.mizool.technology.cassandra;
+
+import javax.enterprise.inject.Disposes;
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.QueryOptions;
+import com.datastax.driver.extras.codecs.jdk8.InstantCodec;
+import com.datastax.driver.extras.codecs.jdk8.LocalDateCodec;
+import com.datastax.driver.extras.codecs.jdk8.LocalTimeCodec;
+import com.google.common.base.Splitter;
+import com.google.common.collect.Iterables;
+
+class ClusterProducer
+{
+    private static final String CASSANDRA_CONTACT_POINTS_PROPERTY_NAME = "mizool.cassandra.contactpoints";
+
+    @Produces
+    @Singleton
+    public Cluster produce()
+    {
+        String addresses = System.getProperty(CASSANDRA_CONTACT_POINTS_PROPERTY_NAME, "127.0.0.1");
```

Comment:
hmm, I'm unsure whether we want to default to localhost.

when admins forget to set this, they will get a "not enough hosts available" exception or the like, and maybe a "configuration exception: no contact points set" is more obvious.

@schroenser, @JanDoerrenhaus, @voons: what do you think?

Reply:
Yes, i had the same thought but decided that the developers have enough properties to remember/setup for the system to work properly.  But let's change it to the exception to be sure it does not lead to magic behavior.